tensorflow/python/kernel_tests/math_ops/segment_reduction_ops_test.py

```diff
@@ -624,7 +626,8 @@ def testValues(self):
                     segment_ids=math_ops.cast(segment_indices,
                                               segment_ids_dtype))
                 tf_ans = self.evaluate(s)
-                self.assertAllClose(np_ans, tf_ans)
+                tol = 1e-6 if dtype != dtypes_lib.bfloat16 else 1e-1
+                self.assertAllClose(np_ans, tf_ans, rtol=tol, atol=tol)
```

Comment:
Same here... `assertAllCloseAccordingToType()` may be better, preferably without needing to adjust the tolerance.